File: app/src/main/java/dev/ragnarok/fenrir/api/model/VkApiProfileInfo.java
package dev.ragnarok.fenrir.api.model;

import com.google.gson.annotations.SerializedName;


public class VkApiProfileInfo {

    @SerializedName("first_name")
    public String first_name;

    @SerializedName("last_name")
    public String last_name;

    @SerializedName("maiden_name")
    public String maiden_name;

    @SerializedName("screen_name")
    public String screen_name;

    @SerializedName("home_town")
    public String home_town;

    @SerializedName("bdate")
    public String bdate;

    @SerializedName("sex")
    public int sex;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/VKApiProfileInfo.kt
package dev.ragnarok.fenrir.api.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class VKApiProfileInfo {
    @SerialName("first_name")
    var first_name: String? = null

    @SerialName("last_name")
    var last_name: String? = null

    @SerialName("maiden_name")
    var maiden_name: String? = null

    @SerialName("screen_name")
    var screen_name: String? = null

    @SerialName("home_town")
    var home_town: String? = null

    @SerialName("bdate")
    var bdate: String? = null

    @SerialName("sex")
    var sex = 0
}